contracts/src/v0.7/dev/KeeperRegistryDev.sol

```diff
@@ -128,77 +171,33 @@ contract KeeperRegistryDev is
   event FundsAdded(uint256 indexed id, address indexed from, uint96 amount);
   event FundsWithdrawn(uint256 indexed id, uint256 amount, address to);
   event OwnerFundsWithdrawn(uint96 amount);
-  event ConfigSet(
-    uint32 paymentPremiumPPB,
-    uint24 blockCountPerTurn,
-    uint32 checkGasLimit,
-    uint24 stalenessSeconds,
-    uint16 gasCeilingMultiplier,
-    uint256 fallbackGasPrice,
-    uint256 fallbackLinkPrice,
-    uint32 maxPerformGas,
-    uint96 minUpkeepSpend
-  );
-  event FlatFeeSet(uint32 flatFeeMicroLink);
+  event UpkeepMigrated(uint256 indexed id, uint256 remainingBalance, address destination);
+  event UpkeepImported(uint256 indexed id, uint256 startingBalance, address importedFrom);
+  event ConfigSet(Config config);
   event KeepersUpdated(address[] keepers, address[] payees);
   event PaymentWithdrawn(address indexed keeper, uint256 indexed amount, address indexed to, address payee);
   event PayeeshipTransferRequested(address indexed keeper, address indexed from, address indexed to);
   event PayeeshipTransferred(address indexed keeper, address indexed from, address indexed to);
   event RegistrarChanged(address indexed from, address indexed to);
   event UpkeepGasLimitSet(uint256 indexed id, uint96 gasLimit);
+  event TranscoderChanged(address indexed from, address indexed to);
 
   /**
    * @param link address of the LINK Token
    * @param linkEthFeed address of the LINK/ETH price feed
    * @param fastGasFeed address of the Fast Gas price feed
-   * @param paymentPremiumPPB payment premium rate oracles receive on top of
```

Comment:
Maybe worth moving this documentation onto the actual struct? It [was pointed out](https://ethereum.stackexchange.com/questions/84265/natspec-commenting-style-for-structs) Compound has started a convention where they use `@member` on structs instead of `@param` because Natspec doesn't support structs yet.